py3status/modules/window_title.py

```diff
@@ -42,20 +45,31 @@ class Py3status:
     format = '{title}'
     max_width = 120
 
+    def _find_focused(self, tree):
+        if isinstance(tree, list):
+            for el in tree:
+                res = self._find_focused(el)
+                if res:
+                    return res
+        elif isinstance(tree, dict):
+            if tree['focused']:
+                return tree
+            return self._find_focused(tree['nodes'] + tree['floating_nodes'])
+        return {}
+
     def window_title(self):
         tree = loads(self.py3.command_output('i3-msg -t get_tree'))
-        window = find_focused(tree)
+        window_properties = self._find_focused(tree).get(
+            'window_properties', {'title': ''})
```

Comment:
`.get('window_properties', {'class': '', 'instance': '', 'title': ''})` is more like what we want but (untested)

```
self.defaults = {x[1:-1]: '' for x in self.py3.get_placeholders_list(self.format)}
```
means we don't care about what placeholders there are.